og.routing.yml

```diff
@@ -0,0 +1,18 @@
+# Routes for Organic groups.
+
+og_ui.subscribe:
+  path: 'group/{entity_type_id}/{entity_id}/subscribe/{field_name}'
+  defaults:
+    _controller: '\Drupal\og\Controller\SubscriptionController::subscribe'
+    _title: 'Join Group'
```

Comment:
This will need to be a callback, as it will be 'Subscribe to group' or 'Request subscription to group` -- according to the group's permissions.